components/TodoDisplay.js

```diff
@@ -0,0 +1,51 @@
+app.component('todo-display', {
+  data () {
+    return {
+      todoItems: []
+    }
+  },
+  template:
+  /* html */
+  `<div class="todo-display">
+    <todo-form @todo-submitted="$_addTodoItem"></todo-form>
+    <todo-list
+      @set-items="$_setTodoItems"
+      @check-item="$_checkItem"
+      @delete-item="$_deleteItem"
+      @update-item="$_updateItem"
+      :todoItems="this.todoItems"
```

Comment:
テンプレートの中では `this` はなくても動きそうな気がしました。

```suggestion
      :todoItems="todoItems"
```